source/brailleDisplayDrivers/hims.py

```diff
@@ -116,6 +116,24 @@ def _get_keys(self):
 		})
 		return keys
 
+
+class BrailleEdge2S(BrailleEdge):
+	"""This device is BrailleEdge which doesn't use hims driver. It only use spp connection.
+	"""
+	usbId = "VID_1A86&PID_55D3"
+
+	def _get_keys(self):

```

Comment:
please add a return type here

```suggestion
	def _get_keys(self) -> dict[str, str]:
```